README.md

```diff
@@ -1,2 +1,242 @@
+
+    `$BALLERINA_HOME/distributions/ballerina-<BALLERINA_VERSION>/bre/security/ballerinaTruststore.p12`
+
+    The default `TRUST_STORE_PASSWORD` is set to "ballerina".
+
+    The `WORK_BOOK_NAME`, `WORK_SHEET_NAME`, and `TABLE_NAME` correspond to the workbook file name (without the .xlsx extension), worksheet name, and table name respectively. Make sure you create a workbook with the same `WORK_BOOK_NAME` as on Microsoft OneDrive before using the connector.
+
+3. Create a new Ballerina project by executing the following command.
+
+	```shell
+	<PROJECT_ROOT_DIRECTORY>$ ballerina new project_name
+	<PROJECT_ROOT_DIRECTORY>$ cd project_name
+	<PROJECT_ROOT_DIRECTORY>$ ballerina add module_name
+	```
+
+4. Import the Microsoft Graph connector to your Ballerina program as follows.
+
+    The following sample program creates a new worksheet on an existing workbook on Microsoft OneDrive. Prior to running this application, create a workbook on your Microsoft OneDrive account with the name "MyShop.xlsx". There needs to be at least one worksheet (i.e., a tab) on the workbook for this sample program to work. 
+
+    The sample application first tries to delete an existing worksheet named "Sales" from the workbook. If its not available, it may throw an error and continue executing the rest of the program. This error will get thrown during the very first round of running the sample application. Make sure that you keep the `ballerina.conf` file with the above-mentioned configuration information before running the sample application.
+
+	```
+    import ballerina/config;
+    import ballerina/log;
+    import ballerina/time;
+    import ballerinax/microsoft.sheets as sheets;
+
+    // Create the Microsoft Graph Client configuration by reading the config file.
+    sheets:MicrosoftGraphConfiguration msGraphConfig = {
+        baseUrl: config:getAsString("MS_BASE_URL"),
+        msInitialAccessToken: config:getAsString("MS_ACCESS_TOKEN"),
+        msClientID: config:getAsString("MS_CLIENT_ID"),
+        msClientSecret: config:getAsString("MS_CLIENT_SECRET"),
+        msRefreshToken: config:getAsString("MS_REFRESH_TOKEN"),
+        msRefreshURL: config:getAsString("MS_REFRESH_URL"),
+        trustStorePath: config:getAsString("TRUST_STORE_PATH"),
+        trustStorePassword: config:getAsString("TRUST_STORE_PASSWORD"),
+        bearerToken: config:getAsString("MS_ACCESS_TOKEN"),
+        clientConfig: {
+            accessToken: config:getAsString("MS_ACCESS_TOKEN"),
+            refreshConfig: {
+                clientId: config:getAsString("MS_CLIENT_ID"),
+                clientSecret: config:getAsString("MS_CLIENT_SECRET"),
+                refreshToken: config:getAsString("MS_REFRESH_TOKEN"),
+                refreshUrl: config:getAsString("MS_REFRESH_URL")
+            }
+        }
+    };
+
+    sheets:MSSpreadsheetClient msSpreadsheetClient = new(msGraphConfig);
```

Comment:
According to the convention abbreviation `MS` for `Microsoft` has to be specified as `Ms` for naming. Therefore, the `sheets:MSSpreadsheetClient` has to be renamed as `sheets:MsSpreadsheetClient`.

Reply:
Done